docs/SnapshotTesting.md

```diff
@@ -121,6 +121,20 @@ You can try out this functionality by cloning the
 [snapshot example](https://github.com/facebook/jest/tree/master/examples/snapshot),
 modifying the `Link` component, and running Jest.
 
+#### Interactive Updates
+
+The snapshots of one test file can be updated in the Jest watch mode:
+
+![](/jest/img/content/interactiveSnapshot.png)
+
+Once you enter the Interactive Snapshot Updates mode, Jest will walk you though
+the failed snapshots of one test file at a time and give you an opportunity to
```

Comment:
I think there's an extra "of" here - should be:

> Jest will walk you through the failed snapshots one test file at a time...

Also `though` is a typo, should be `through`